### ISSUE TITLE
Normative Text in Examples & Notes

### ISSUE THREAD

**Issue by KathiSchleidt** [opengeospatial/om-swg#186]
**-029 EXAMPLE	A provider may only serve information**
EXAMPLEs cannot contain the verbal forms "shall", "should" or "may". Please review the verbal forms used in this Example. 	

**-072 EXAMPLE	In order to explicitly describe the capabilities of an Environmental Monitoring Facility,**
one must provide information on what Observable Properties are being measured with which methodology.	EXAMPLEs cannot contain the verbal form "must". Please rephrase by using statement of fact. 
Note that in accordance with the ISO House Style, the use of personal pronouns should also be avoided (in this case, "one"). 
A possible rephrasing of this sentence could be:
"..... information needs to be provided on what Obersable Properties..."	
Pertains to 10.5.2

**Comment by KathiSchleidt** [comment]
**-036 may can potentially**	
Note that "may" denotes permission, whereas "can" denotes possibility. Please review use of "may" and "can" throughout.

**Comment by KathiSchleidt** [comment]
**-038 remote sensing observation might obtain the reflectance colour**	
The word "might" is not recommended as it is a little ambiguous. It is suggested to use the verbal form "can" or "can potentially" instead of "might". 	

KS: damned if you do, damned if you don't! ;)

**Comment by KathiSchleidt** [comment]
**-045 NOTE on Unitless**
In the case where the result of the Observation is a classification, for which no unit exists, the UoM should be declared as unitless 	
NOTEs cannot contain the verbal forms "may", "should" or "shall". Please rephrase.

**Comment by KathiSchleidt** [comment]
**-050 Normative text in Notes**
NOTE 1	The result can be of Any type as it may represent the value of any feature property.
NOTE 2	If the observed property is a spatial operation or function, the type of the result may be a coverage.	
NOTEs cannot contain the verbal forms "may", "should" or "shall". Please rephrase.	

Pertains to 8.2.8

**Comment by KathiSchleidt** [comment]
**-105 Refactoring of the domain models may be required to separate the ultimate and proximate features of interest**
Please check the verbal form used here ("may" or "can")	

Section C.4.3. (informative, so seeing as an example)

**Comment by KathiSchleidt** [comment]
**-110 might**
Please avoid the use of "might", as it is quite ambiguous in terms of meaning. Please try to use "can" (possibility) or "may" (permission) instead, or "can potentially" in the case of a hypothetical situation.

**Comment by KathiSchleidt** [comment]
**-029** EXAMPLE A provider may only serve information
Rephrase as "Some providers only serve information..."

**-072**
one must provide information on what Observable Properties are being measured with which methodology.
change to
information on what Observable Properties are being measured with which methodology is provided.

**-036 may can potentially**
OUCH - 104 "may" in the doc
replace with "can" outside reqs except:
7.2.2.1	Introduction: replace "may not be" with "is not always"
7.2.3	Role of Sample: replace "the sampled object may not be known" with "the sampled object **is** not known"
7.3.2	Relationship between Sample and domain features (under Fig 6): replace "Sample and the Domain feature may appear as" with "Sample and the Domain feature **can potentially** appear as "

**-038** replace with "might" with "can" or "can potentially"

**Comment by KathiSchleidt** [comment]
**-045 NOTE  8.2.16	Constraint unit of measure**
`the UoM should be declared as unitless`
replace should with can

**-050 Normative text in Notes 8.2.8	Association result**
```
NOTE 1	The result can be of Any type as it may represent the value of any feature property.
NOTE 2	If the observed property is a spatial operation or function, the type of the result may be a coverage. 
```
Replace may with can

**-057 NOTE 9.1.2	Association metadata**
`Attention should be given not to reinvent semantic that is explicitly modelled in the OMS model.`	
NOTEs cannot contain the verbal form "should". Please rephrase.

**This one is tricky, I'm torn between deleting the note, converting to body text and converting to a rec**

**-067 NOTE 9.9.3	Attribute name**	
`The value of the name should be taken from a well-governed source if possible.`	
NOTEs cannot contain the verbal form "should". Please rephrase.	

**This one is tricky, I'm torn between deleting the note, converting to body text and converting to a rec**
Proposal: Using well-governed sources for the value of the name enhances reusability. 

**-068 NOTE 9.9.4	Attribute value**
The type “Any” should be substituted by a suitable concrete type, such as CI_ResponsibleParty or Measure.	
NOTEs cannot contain the verbal form "should". Please rephrase.	

Proposal: In concrete realizations, the type "Any" can be substituted...

**-082 NOTE 12.2.3	Attribute parameter**	
Parameter should NOT be utilized to 	
NOTEs cannot contain the verbal forms "shall", "should" or "may". Please rephrase or convert to body text.	

Convert to Body Text?

**-084 NOTE 12.3.4	Attribute parameter**	
Parameter should NOT 	
NOTEs cannot contain the verbal forms "shall", "should" or "may". Please rephrase or convert to body text.

Convert to Body Text?

**Comment by KathiSchleidt** [comment]
**-105** 
Refactoring of the domain models may be required to separate the ultimate and proximate features of interest
Please check the verbal form used here ("may" or "can")

change to can

**-110 might**
Please avoid the use of "might", as it is quite ambiguous in terms of meaning. Please try to use "can" (possibility) or "may" (permission) instead, or "can potentially" in the case of a hypothetical situation.

replace with can

**Comment by KathiSchleidt** [comment]
Agreed

**Comment by sgrellet** [comment]
**-036 may can potentially**
Note that "may" denotes permission, whereas "can" denotes possibility. Please review use of "may" and "can" throughout.

may -> can change added for 13.5.3	Attribute classification

**-050 Normative text in Notes**
@KathiSchleidt  : I just have an issue with this one.
    we still have '_NOTE 1	The result can be of Any type as it may represent the value of any feature property_.'
I understand that "may denotes a possibility"
but still we do have this rule "NOTEs cannot contain the verbal forms "may", "should" or "shall". Please rephrase."

=> how do we do with this one ?

**Comment by KathiSchleidt** [comment]
Checked the new fix under 13.5.3 Attribute classification, OK

**-050 Normative text in Notes**
Changed the Note to read:

`NOTE 1 The result can be of Any type as it can represent the value of any feature property.`

Please counter-check!

**Comment by sgrellet** [comment]
Looks good to me
